generators/generator-base.js

```diff
@@ -1451,6 +1451,17 @@ module.exports = class extends PrivateBase {
         return joinTableName;
     }
 
+    generateTableJoins(relationships) {
```

Comment:
Could you move it in `entity-server` subgen, prefixed with `_` so it can be used in templates?